packages/compiler/core/types.ts

```diff
@@ -346,29 +375,39 @@ export interface SymbolTable extends Map<string, Sym> {
 
 // prettier-ignore
 export const enum SymbolFlags {
```

Comment:
Was this formatting on purpose?

Reply:
No, and I'm mad about it. I never touched the comment but something I did caused my beautiful formatting to break 😭